File: cwd/conf.lua
local love = require("love")
function love.conf(t)
	t.identity              = "livesim4"
	t.version               = "0.10.2"
	t.console               = false
	t.accelerometerjoystick = false
	t.externalstorage       = true
	t.gammacorrect          = false
	
	t.window.title          = "FiLive!"
	t.window.icon           = nil
	t.window.width          = 960
	t.window.height         = 640
	t.window.borderless     = false
	t.window.resizable      = true
	t.window.minwidth       = 320
	t.window.minheight      = 240
	t.window.fullscreen     = love._os == "iOS"
	t.window.fullscreentype = "desktop"
	t.window.vsync          = true
	t.window.msaa           = 0
	t.window.display        = 1
	t.window.highdpi        = false
	t.window.x              = nil
	t.window.y              = nil
	
	t.modules.joystick      = false
	t.modules.physics       = false
end

File: src/conf.cpp
// This is equivalent to love.conf
// Edit if necessary. Default is supplied.
#include "Boot.h"

GameWindowSettings ws = {
	// fullscreen
	false,
	// fstype
	love::window::Window::FULLSCREEN_DESKTOP,
	// vsync
	1,
	// msaa
	0,
	// stencil
	true,
	// depth
	0,
	// resizable
	true,
	// minwidth
	1,
	// minheight
	1,
	// borderless
	false,
	// centered
	true,
	// display
	0,
	// highdpi
	false,
	// refreshrate: keep this 0!
	0.0,
	// useposition, x, y
	false, 0, 0,
	// width,height
	960, 640,
	// title
	"livesim4",
	// icon
	""
};

// It must be named "conf"
GameConfiguration conf = {
	// identity
	"livesim4",
	// appendidentity
	true,
	// accelerometerjoystick
	false,
	// externalstorage
	true,
	// gammacorrect
	false,
	// mixwithsystem
	true,
	// window. Set to `nullptr` to defer window creation.
	&ws,
	// modules
	// Modules which are always unconditionally enabled:
	// love.data
	// love.filesystem
	// love.thread
	std::vector<std::string>({
		"audio", "data", "event", "font", "graphics", "image",
		"joystick", "keyboard", "math", "mouse", "physics", "sound",
		"timer", "touch", "video", "window"
	})
};
